use_case_examples/cifar/README.md

```diff
@@ -0,0 +1,55 @@
+# CIFAR-10 and CIFAR-100 Classification with FHE
+
+This repository provides resources and documentation on different use-cases for classifying CIFAR-10 and CIFAR-100 images using Fully Homomorphic Encryption (FHE). Each use-case demonstrates different techniques and adaptations to work within the constraints of FHE.
+
+## Table of Contents
+1. [Use-Cases](#use-cases)
+   - [Fine-Tuning CIFAR-10/100](#fine-tuning-cifar-10100)
+   - [Training Ternary VGG on CIFAR10](#training-ternary-vgg-on-cifar10)
+   - [CIFAR-10 with a Split Clear/FHE Model](#cifar-10-with-a-split-clearfhe-model)
+2. [Installation](#installation)
+3. [Further Reading & Resources](#further-reading--resources)
+
+## Use cases
+
+### Fine-Tuning CIFAR-10/100
+
+- **Description**: This use-case explores how to convert a pre-trained CNN (on imagenet) to its FHE equivalent using QAT and Concrete ML. The conversion process involves adapting a VGG11 network and quantizing the network for FHE.
```

Comment:
maybe say "Quantize Aware Training (QAT)" instead of just "QAT" here ? Or even reference the doc ?

Reply:
Yes agreed.